.github/workflows/integrate.yaml

```diff
@@ -56,8 +56,9 @@ jobs:
     strategy:
       matrix:
         microk8s-versions:
-          - 1.27-strict/stable
+          - 1.29-strict/stable
           - 1.30-strict/stable
+          - 1.31-strict/stable
```

Comment:
Can we keep it to 2 versions only?

Reply:
yes, we keep the lowest and highest supported versions. On the contrary, we don't have to test o11y with more than one versions. That was implemented by accident.